server/src/utils/validationUtil.ts

```diff
@@ -0,0 +1,21 @@
+export function validateEmail(email: string) {
```

Comment:
this should be under a shared folder, since you have the same function on backend and frontend aswell